pkgs/tools/networking/ht-rust/default.nix

```
@@ -0,0 +1,32 @@
+{ pkgs, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  name = "ht-rs-${version}";
+  version = "0.4.0";
+  src = fetchFromGitHub {
+    owner = "ducaale";
+    repo = "ht";
+    rev = "v${version}";
+    sha256 = "jT4Y6Yr3Cs2YPDSPNxTdIbyl6dP8v+HqG2zQJnPHvHA=";
+  };
+
+  cargoSha256 = "sha256-GTH7rXbCWLl/R1xUdA5dYgx592rw5ITQuqDoNoXCL2g=";
+
+  cargoPatches = [
+    # a patch file to change package & output binary name as "ht" conflicts with another package
+    ./bin-rename.patch
+  ];
+
+  doInstallCheck = true;
+  postInstallCheck = ''
+    $out/bin/ht-rs --help > /dev/null
+  '';
+
+  meta = with pkgs.lib; {
```

```suggestion
  meta = with lib; {
```

and import `lib` instead of `pkgs`.

---

Ditto